remote-trait-object/src/context.rs

```diff
@@ -70,7 +95,8 @@ impl Config {
         Self {
             name: "my rto".to_owned(),
             call_slots: 512,
-            call_timeout: std::time::Duration::from_millis(1000),
+            maximum_services_num: 65536,
```

Comment:
Isn't it more common to 65535 as a maximum value rather than 65536?
I thought 65535 is better because it is u16::MAX https://doc.rust-lang.org/std/primitive.u16.html#associatedconstant.MAX
It is not an important comment. I want to know in which point 65536 is better.

Reply:
That makes sense. I'll fix

Reply:
Well I thought about it, but it's common to use 2^N instead of 2^N-1 for a _size_ of something, not _index_.